tutoraspects/plugin.py

```diff
@@ -255,7 +255,14 @@
                 "student": "Student",
             },
         ),
+        (
+            "SUPERSET_EMBEDDABLE_DASHBOARDS",
+            {
+                "instructor-dashboard": "1d6bf904-f53f-47fd-b1c9-6cd7e284d286",
+            },
+        ),
         ("SUPERSET_ADMIN_EMAIL", "admin@openedx.org"),
+        ("SUPERSET_LMS_EMAIL", "lms@openedx.org"),
```

Comment:
I'm thinking that we should probably change these to invalid email addresses by default so no one gets accidentally emailed private questions about sites. Asking around Axim right now for ideas.

Reply:
We can update the default to be lms-admin@{{lms_host}}

Reply:
I think we can just go with `superset/lms-admin@aspects.invalid` since it will never resolve and cause errors if anyone tries to use it.

Reply:
Ready